event_scheduler/event_scheduler.py

```diff
@@ -4,10 +4,12 @@
 import sys
 import threading
 
+_sentinel = object()
```

Comment:
thats pretty cool where did you learn about `object()` ?

Reply:
It was in the sched, I realized I was putting a mutable as a default argument which is really bad so I had to change it. If you use a mutable as a default argument and you modify it, the default argument will change too.
```python
def foo(kwargs={}):
    print(kwargs)
    kwargs['something'] = 0


foo()
foo()
```
\>\>\> {}
\>\>\> {'something' : 0}